src/database/sql_database.cc

```diff
@@ -2539,8 +2538,7 @@ bool SQLDatabase::doMetadataMigration()
 
 void SQLDatabase::migrateMetadata(int objectId, const std::string& metadataStr)
 {
-    std::map<std::string, std::string> dict;
-    dictDecode(metadataStr, dict);
+    std::map<std::string, std::string> dict = dictDecode(metadataStr);
```

Comment:
Wonder if auto is useful here.

Reply:
Yes, we could. But since it was there before, I'd like to keep it because it increases readability of the code. This way, we look at it and know its a map of string->string.